show/bgp_frr_v4.py

```diff
@@ -102,10 +111,16 @@ def neighbors(ipaddress, info_type, namespace):
 def network(ipaddress, info_type, namespace):
     """Show IP (IPv4) BGP network"""
 
-    if multi_asic.is_multi_asic() and namespace not in multi_asic.get_namespace_list():
-        ctx = click.get_current_context()
-        ctx.fail('-n/--namespace option required. provide namespace from list {}'\
-            .format(multi_asic.get_namespace_list()))
+    namespace = namespace.strip()
+    if multi_asic.is_multi_asic():
```

Comment:
I know you are just keeping original logic here, that if it is multi-asic, and namespace is not provided, error out.
but I think original logic may not be updated, can you change to: if multi asic, and namespace is not provided, print for all asics?
refer to: https://github.com/sonic-net/sonic-utilities/pull/2427
another pros for this is, you are running this cmd on sup, if one chassis has both multi/single-asic LCs, `show ip bgp sum` will output for all LCs, instead of having multi-asic LC error out, in this case we could have at least cmd working for both multi/single-asic.